src/stdlib/date.md

```diff
@@ -81,7 +86,12 @@ The Date API is still preliminary with only a subset of properties and methods i
   function setUTCDate(value: i32): i32
   ```
   Sets the day of the month according to universal time.
-  
+
+* ```ts
+  function getUTCDay(): i32
+  ```
+  Gets the day of the week in the specified date according to universal time, where 0 represents Sunday.
```

Comment:
```suggestion
  Returns the day of the week in the specified date according to universal time, where 0 represents Sunday.
```

Reply:
I just try to follow already existing conventions which use `Gets` and `Sets` as starting word for rest of set/set methods.

Reply:
No strong opinions, looks good to me either way 👍  I was just following https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCDay

Reply:
yeah, I also use MDN as reference.